data_write/data_write.py

```diff
@@ -974,16 +1045,17 @@ def write_tx_data():
 
 
 
-
         parameters_holder = {}
         for meta in integration_meta.sequences:
             for rx_freq in meta.rxchannel:
                 parameters = DATA_TEMPLATE.copy()
                 parameters['borealis_git_hash'] = self.git_hash.decode('utf-8')
-
                 parameters['timestamp_of_write'] = (write_time - epoch).total_seconds()
-                parameters['experiment_id'] = np.uint32(integration_meta.experiment_id)
-                parameters['experiment_string'] = integration_meta.experiment_string
+                parameters['experiment_id'] = np.int64(integration_meta.experiment_id)
+                parameters['experiment_name'] = integration_meta.experiment_name
+                parameters['experiment_comment'] = integration_meta.experiment_comment  
+                parameters['slice_comment'] = rx_freq.slice_comment
+                parameters['num_slices'] = len(meta.rxchannel)
```

Comment:
Shouldnt this be len(integration_meta.sequences) * len(meta.rxchannel)?

Reply:
yes thanks!